src/main/java/net/imglib2/algorithm/morphology/table2d/Spur.java

```diff
@@ -73,7 +73,7 @@
 
 		final ImgFactory< T > factory;
 		if ( extendedVal instanceof NativeType )
-			factory = ( ImgFactory< T > ) Util.getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
+			factory = Util.< NativeType >getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
```

Comment:
@hanslovsky @axtimwalde For me this is a compile error in Eclipse. Strangely it is not for mvn on the command line. In this case, I have to say, I agree with Eclipse: `ImgFactory<NativeType> != ImgFactory<T>`. I wonder why this compiles. Which Eclipse version are you using?